src/components/WorkOrder/WorkOrderHeader.js

```diff
@@ -55,6 +65,16 @@ const WorkOrderHeader = ({
         <div className="lbh-body-xs">
           <span>Assigned to: {workOrder.owner}</span>
         </div>
+        {workOrder.closedDated && (
+          <div className="lbh-body-xs">
+            <span>
+              <strong>
+                {completionReason()}:{' '}
+                {formatDateTime(new Date(workOrder.closedDated))}
```

Comment:
How about combine these 2 lines together, like that:
```
{`${completionReason()}:  ${formatDateTime(new Date(workOrder.closedDated))}`}
```

Reply:
{''} was added automatically with prettier:fix